### ISSUE TITLE
A11y: Error messages from input fields are not programmatically linked to the input fields

### ISSUE THREAD

**Issue by blacksheep--** [shopware/platform#7036]
*PHP Version:* 8.2   | *Shopware Version:* 6.6.9.0  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

In the DOM, a specific error message such as “Last name must not be empty” is displayed after the incorrectly completed input fields. This error message is not programmatically linked to the input field.

As a result, screen reader users do not receive any information about this error message when they tab into the input field.

*_Expected behaviour:_*

The error message must be linked to the corresponding incorrect input field using the `aria-describedby` attribute.

*_How to reproduce:_*

Enable `ACCESSIBILITY_TWEAKS` Feature-Flag

1. navigate to /account/profile
2. remove input of any required field (e.g. firstname)
3. leave the field

![Image](https://github.com/user-attachments/assets/0db29d1c-d3be-467e-a35e-30ad54c46a61)

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40810**



[created from NEXT-40810, comment 541342]

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40810, comment 541562]

**Comment by Phil23** [comment]
See my other comment: https://github.com/shopware/shopware/issues/7030#issuecomment-2696632519
